browser-test/src/admin_api_keys.test.ts

```diff
@@ -35,13 +35,23 @@ describe('Managing API keys', () => {
     const credentials = await adminApiKeys.createApiKey({
       name: 'Test API key',
       expiration: '2022-01-31',
-      subnet: '8.8.8.8/32',
+      subnet: '0.0.0.0/0',
       programSlugs: ['api-using-program'],
     })
 
     expect(typeof credentials).toEqual('string')
 
     await adminApiKeys.expectApiKeyIsActive('Test API key')
+
+    let apiResponse = await adminApiKeys.callCheckAuth(credentials)
+    expect(apiResponse.status).toEqual(200)
+    await adminApiKeys.expectKeyCallCount('test-api-key', 1)
```

Comment:
Blocking: Since the key count is updated from a background thread asynchronously, it's possible that we could run into an instance where the call count hasn't been updated by the time we arrive at `expectKeyCallCount`. This could lead to a low-level of flake.

Given the expected asynchronous nature, I'd suggest converting the `expectKeyCallCount` / `expectLastCallIpAddressToBeSet` methods to poll (or wrap the expectation in code that polls until the desired state is met).

Reply:
done